packages/next-server/server/config.ts

```diff
@@ -120,10 +120,12 @@ export default function loadConfig(
     }
 
     if (
-      userConfig.target === 'serverless' &&
+      userConfig.target != null &&
```

Comment:
```suggestion
      userConfig.target &&
```

Could just make sure it's truthy here